### ISSUE TITLE
SteamAxe on grass, Crash

### ISSUE THREAD

**Issue by Divineaspect** [Esteemed-Innovation/Flaxbeards-Steam-Power#392]
## Bug description
Steam axe with void  upgrade and lumber chain.

## Base information
* Minecraft version:1.7.10
* Minecraft Forge version:
* Flaxbeard's Steam Power version:0.29.2

## Crash report
```
client: http://pastebin.com/zSmwCjpD
server: http://pastebin.com/tuSmuhTP
```

**Comment by Divineaspect** [comment]
Occurs with drill on grass as well.
Clarification, tall grass.

**Comment by elifoster** [comment]
Fixed by the same thing that fixed #385
